src/app/api/template/route.ts

```diff
@@ -0,0 +1,60 @@
+import { NextRequest } from 'next/server';
+import {
+  StatusCodeType,
+  errorResponse,
+  successResponse,
+} from '@/lib/response/responseUtil';
+import { prisma } from '@/lib/prisma';
+import { handleCommonErrors } from '@/lib/response/errorUtil';
+import { getServerAuthSession } from '@/lib/auth';
+import { isTrainerOrAdmin, hasTemplateDeleteRights } from '@/lib/auth-utils';
+import { translator } from '@/lib/i18n';
+import { templateDeleteSchema } from '@/lib/zod/templates';
+
+export async function DELETE(request: NextRequest) {
+  console.log('template delete');
```

Comment:
Debugging console.log call needs to be removed

Reply:
Fixed